doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -461,6 +463,41 @@ If the configuration is successfully downloaded, the system asks for the followi
 * Security server code (**reference data: 2.4**), which is chosen by the security server administrator and which has to be unique across all the security servers belonging to the same X-Road member.
 * Software token’s PIN (**reference data: 2.5**). The PIN will be used to protect the keys stored in the software token. The PIN must be stored in a secure place, because it will be no longer possible to use or recover the private keys in the token once the PIN has been lost.
 
+### 3.4 Configuring configuration backup encryption
+
+It is possible to automatically encrypt security server configuration backups. Security server uses The GNU Privacy Guard (https://www.gnupg.org)
+for backup encryption and verification. Backups are always signed, but backup encryption is initially turned off.
+To turn encryption on, please override the default configuration in the file `/etc/xroad/conf.d/local.ini`, in the `[proxy]` section.
+(add or edit this section) 
+
+    [proxy]
+    backup-encrypted=true
+    backup-public-key-path=/etc/xroad/backupkeys
+
+To turn backup encryption on, please change the backup-encrypted property value to true.
+By default, additional encryption keys are stored in the /etc/xroad/backupkeys directory.
+The default directory can be changed by modifying the backup-public-key-path property value.
```

Comment:
Please note the code blocks on `backup-encrypted`, `/etc/xroad/backupkeys` and `backup-public-key-path`:

> To turn backup encryption on, please change the `backup-encrypted` property value to `true`. By default, additional encryption keys are stored in the `/etc/xroad/backupkeys` directory. The default directory can be changed by modifying the `backup-public-key-path` property value.